.rubocop.yml

```diff
@@ -8,7 +8,7 @@ AllCops:
   DisplayCopNames: true
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/218261244-33862ea3-a21b-4a0f-8831-34b0b1179d54.png)

- [ ] I hope these are the key features of this project :smile: Kindly put OOP School Library key features in this section :blush: